language/en.js

```diff
@@ -28,6 +28,22 @@ H5PEditor.language.core = {
   replaceImage: 'Replace Image?',
   confirmReplaceImage: 'Are you sure you want to replace this image?',
   onlyPasteImageFiles: 'You may only paste image files.',
+  uploadVideo: 'Upload Video',
+  uploadAudio: 'Upload Audio',
+  uploadOr: 'or',
+  dragAndDropAndPasteVideo: 'Drag and drop video file here to upload or paste by',
+  dragAndDropAndPasteAudio: 'Drag and drop audio file here to upload or paste by',
+  pasteVideoLink: 'Paste YouTube link or other video source',
+  pasteAudioLink: 'Paste link or other audio source URL',
+  dropVideo: "Drop video file here to upload",
+  dropAudio: "Drop audio file here to upload",
+  enterVideoLink: "Enter a video link",
+  enterAudioLink: "Enter an audio link",
```

Comment:
```suggestion
  enterVideoLink: 'Enter a video link',
  enterAudioLink: 'Enter an audio link',
```

Reply:
You're spell-checking now? :-)

Reply:
Heheh I'm reading through most of the PRs in case I need to implement types for changes and stumbled upon this.

Reply:
I check all of them for interesting changes, too - just suggested to not hardcode "ctrl" but to check for MacOS and then use "command" last week - but I leave quotation marks to linters ... 😅

Reply:
That's a good catch!! Without CI I assume linting is the responsibility of humans

Reply:
Wouldn't even have to be CI, but yes, a proper config so the IDE complains at least feels sensible.

Reply:
Hehe nice catch :). Yeah, we really should prioritize getting ESLint in to avoid issues like these 😅. We did recently add Webpack so should be straight forward to e.g. add lint to the build/production command.